File: test4.py
#!/usr/bin/python
from array import *
import random
import time
count = 0
array = []
while (count < 100000): 
	a = random.randint(0, 100000)
	b = random.randint(0, 100000)
	c = random.randint(0, 100000)
	var = str(a) + str(b) + str(c)
	array.append(int(var))
	count = count + 1

a = time.time()
average = sum(array) / float(len(array))
print time.time() - a

File: test4.rb
#!/usr/bin/ruby
require 'securerandom'
i = 1
sample = []
while i < 100000
	a = SecureRandom.random_number(100000)
	b = SecureRandom.random_number(100000)
	c = SecureRandom.random_number(100000)
	num = a.to_s + b.to_s + c.to_s
	num = num.to_i
	sample.push num
	i += 1
end

beginning = Time.now

average = sample.reduce(:+).to_f / sample.size

puts Time.now - beginning



